bin/compare_api_surfaces.py

```
@@ -0,0 +1,356 @@
+    if export_blocking:
+        all_blocking.extend(export_blocking)
+
+    if all_informational:
+        print("\n".join(all_informational))
+        print()
+
+    if all_blocking:
+        print("\n".join(all_blocking))
+        print("\nBREAKING API changes detected vs master!")
+        return 1
+    else:
```

Ruff (0.15.7)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)